kits/survey/package.json

```diff
@@ -3,22 +3,41 @@
   "version": "1.0.0",
   "description": "Kit for conducting non-binding token signaling votes",
   "main": "index.js",
-  "directories": {
-    "test": "test"
-  },
   "scripts": {
-    "migrate": "truffle migrate --network rpc --reset"
+    "prepublishOnly": "truffle compile && npm run abi:extract -- --no-compile",
+    "abi:extract": "truffle-extract --output abi/ --keys abi",
+    "deploy:rinkeby": "truffle exec --network rinkeby scripts/deploy.js",
+    "deploy:rpc": "truffle exec --network rpc scripts/deploy.js",
+    "deploy:devnet": "truffle exec --network devnet scripts/deploy.js",
+    "migrate": "truffle migrate --network rpc --reset",
+    "publish:rpc": "aragon apm publish major $(npm run deploy:kit:rpc | tail -n 1) --environment default",
+    "deploy:kit:rpc": "aragon contracts compile --all && aragon contracts exec --network rpc scripts/survey-deploy.js",
+    "publish:staging": "aragon apm publish major $(npm run deploy:kit:staging | tail -n 1) --environment staging",
+    "deploy:kit:staging": "aragon contracts compile --all && ENS=0xfe03625ea880a8cba336f9b5ad6e15b0a3b5a939 aragon contracts exec --network rinkeby scripts/survey-deploy.js",
+    "publish:rinkeby": "aragon apm publish major $(npm run deploy:kit:rinkeby | tail -n 1) --environment rinkeby",
+    "deploy:kit:rinkeby": "aragon contracts compile --all && ENS=0x98Df287B6C145399Aaa709692c8D308357bC085D aragon contracts exec --network rinkeby scripts/survey-deploy.js",
+    "publish:mainnet": "aragon apm publish major $(npm run deploy:kit:mainnet | tail -n 1) --environment mainnet",
+    "deploy:kit:mainnet": "aragon contracts compile --all && ENS=0x314159265dd8dbb310642f98f50c066173c1259b aragon contracts exec --network mainnet scripts/survey-deploy.js"
   },
+  "files": [
+    "abi/",
+    "build/",
+    "contracts/",
+    "scripts/",
+    "truffle.js"
+  ],
   "author": "Aragon Association <legal@aragon.org>",
   "license": "GPL-3.0-or-later",
   "devDependencies": {
+    "@aragon/cli": "^5.2.0",
     "dotenv": "^6.1.0",
     "eth-ens-namehash": "^2.0.8",
-    "truffle": "4.1.14"
+    "truffle": "4.1.14",
+    "truffle-extract": "^1.2.1"
   },
   "dependencies": {
     "@aragon/apps-shared-minime": "^1.0.0",
-    "@aragon/apps-survey": "^1.0.0-beta.1",
+    "@aragon/apps-survey": "^1.0.0-rc.3",
```

Comment:
I see this is the last version published in npmjs, but we already have `1.0.0` on `aragon-apps` master branch. Should we update npmjs?